helm_deploy/values-dev.yaml

```diff
@@ -22,6 +22,7 @@ generic-service:
     CIAG_INDUCTION_UI_URL: "http://ciag-induction-dev.hmpps.service.justice.gov.uk"
     DPS_URL: "https://digital-dev.prison.service.justice.gov.uk"
     FRONTEND_COMPONENT_API_URL: "https://frontend-components-dev.hmpps.service.justice.gov.uk"
+    FEEDBACK_URL: "https://eu.surveymonkey.com/r/H9CYM7B"
```

Comment:
This is the right idea , but instead of defining it in each environment helm chart you can do it once in the "top level / shared" helm chart
`helm_deploy/hmpps-education-and-work-plan-ui/values.yaml`